Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -62,40 +66,71 @@ namespace AzTestRunner
             return INCORRECT_USAGE;
         }
 
+        // copy over the command line args to a vector so we can expand with any arguments from file
+        std::vector<std::string> arguments(argv, argv + argc);
+
         // capture positional arguments
         // [0] is the program name
-        std::string lib = argv[1];
-        std::string symbol = argv[2];
+        std::string lib = arguments[1];
+        std::string symbol = arguments[2];
```

Comment:
These can be changed to std::string_view
```suggestion
        std::string_view lib = arguments[1];
        std::string_view symbol = arguments[2];
```

Reply:
Fixed 👍

Reply:
These args get popped off of the args list by the arg parsing loop so I couldn't use a string view here.